web/src/components/Chat.tsx

```diff
@@ -27,6 +28,20 @@ const Container = styled(Stack)(({ theme }) => ({
   },
 })) as typeof Stack
 
+const StyledLeft = styled(Box)`
+  display: flex;
+  justify-content: flex-end;
+  gap: 6px;
+  width: 100%;
+`
+
+const StyledRight = styled(Box)`
+  display: flex;
+  justify-content: flex-start;
+  gap: 6px;
+  width: 100%;
+`
```

Comment:
🤔  Is this naming correct? `flex-start` should be `left` and `flex-end` `right`, shouldn't it?